### PR TITLE
make dbt-bigquery python model easier to use

### PR DESCRIPTION
**Pull request by wazi55**
[Re:] https://github.com/dbt-labs/docs.getdbt.com/pull/4150

## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `website/static/_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/build/python-models.md
Comment (dataders) [inline website/docs/docs/build/python-models.md]:
```suggestion
**Additional setup**: The user or role should have the adequate IAM permission to be able to trigger a job through Dataproc Cluster or Dataproc Serverless
```

File: website/docs/docs/build/python-models.md
Comment (dataders) [inline website/docs/docs/build/python-models.md:699]: I assumed that this detailed information would live on somewhere else, like the new section you created within `bigquery-configs`?

File: website/docs/docs/build/python-models.md
Comment (dataders) [inline website/docs/docs/build/python-models.md]: see [our style guidance on link formatting/naming](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md#link-format)

File: website/docs/docs/build/python-models.md
Comment (dataders) [inline website/docs/docs/build/python-models.md]: perhaps something like this table is neater than what you have? and involves less repeated text?

| submission method | `cluster`                                                                                                                                                                                                                       | `serverless`                                                                                                                                                   |
| ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| notes             | can be pre-existing cluster. can also be used to create clusters                                                                                                                                                                | Dataproc Serverless does not require a ready cluster, but it can also mean the cluster is slower to start.                                                     |
| packages          | you can add third-party packages while creating the cluster with the [Spark BigQuery connector initialization action](https://github.com/GoogleCloudDataproc/initialization-actions/tree/master/connectors#bigquery-connectors) | build your own [custom container image](https://cloud.google.com/dataproc-serverless/docs/guides/custom-containers#python_packages) with the packages you need |

**Review by dataders — CHANGES_REQUESTED** [review]
Thanks @wazi55! These docs have needed some TLC and your contributions are certainly in the right direction. That said
1. I think there's some polishing needed, and
2. I'm not confident as to exactly what "good enough" is here, and would defer to the dbt-bigquery community

here's direct links to preview pages that I'd recommend having your colleagues (and the community) review for content and readability.

- [Python models: Supported Data Platforms (BigQuery)](https://docs-getdbt-com-git-fork-wazi55-wazi55-dbt-bigq-89e919-dbt-labs.vercel.app/docs/build/python-models#specific-data-platforms)
- [Bigquery Configuration: Submitting a Python model](https://docs-getdbt-com-git-fork-wazi55-wazi55-dbt-bigq-89e919-dbt-labs.vercel.app/reference/resource-configs/bigquery-configs#submitting-a-python-model)

Once you get sign off, I can ask our product docs team to do a copy review

**Comment by mirnawong1** [comment]
hey @wazi55 - thanks for opening this pr! Anders provided some feedback, wanted to check if you had any add'l questions?

**Comment by matthewshaver** [comment]
Been over 8 months since this had any movement from the author. Closing due to age but we can re-open if this is something we need to visit again.

**Pull request closed without merging**
